recipes/adorn/LICENSE.rst

```diff
@@ -0,0 +1,203 @@
+Apache License
```

Comment:
Since `LICENSE.rst` is already included in the PyPI tarball, you don't need to (and shouldn't) include it here.

Reply:
Thank you @dopplershift I have made both changes and the checks are now passing.